tern/formats/spdx/spdxtagvalue/layer_helpers.py

```diff
@@ -101,7 +140,6 @@ def get_layer_block(layer_obj, template, image_loc=''):
     would be downloaded along with the image.'''
     block = ''
     mapping = layer_obj.to_dict(template)
-    layer_checksum = get_layer_checksum(layer_obj)
```

Comment:
This is not a blocker to merge, just curious on rationale: Why delete this line and variable assignment here if we are replacing it instead with two function calls to `get_layer_checksum` and still creating the layer_checksum variable inside `get_layer_file_data_block`?

Reply:
The layer's checksum is used to differentiate the same file existing in multiple layers (particularly in the case of BerkleyDB files which get copied over to the next layer if any one of the linked files are modified). So for `get_layer_file_data_block` we need the layer's checksum in a couple of places to generate the SPDXID of the file. Meanwhile, we still need the layer's checksum for describing it at the layer level, hence just calling `get_layer_checksum` in line 164 below.

Reply:
The `layer_checksum` we get in `get_layer_file_data_block` will be the same as `get_layer_checksum(layer_obj)` passed in to `block += 'PackageChecksum: {}\n'.format(get_layer_checksum(layer_obj))` right?

Reply:
Yup, same one. It just needs to be used in the `get_file_block` so rather than call it again, I stored the value in a temporary variable.